examples/rotating-cube/main.zig

```diff
@@ -158,7 +159,7 @@ pub fn update(app: *App) !bool {
         const ubo = UniformBufferObject{
             .mat = zm.transpose(mvp),
         };
-        encoder.writeBuffer(app.uniform_buffer, 0, &[_]UniformBufferObject{ubo});
+        queue.writeBuffer(app.uniform_buffer, 0, &[_]UniformBufferObject{ubo});
```

Comment:
We may want to leave this as-is?  Although it's fine with me to change as other examples exercise the encoder writeBuffer method.

Reply:
it's a dawn extension and should be avoided as we don't know if they are going to keep it

Reply:
Interesting, I didn't know this. Which one is the extension?

Reply:
`CommandEncoder.writeBuffer`